frontend/src/pages/CourseSelector/CourseMenu/CourseTitle/CourseTitle.tsx

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
 import {
-  LockOutlined, MinusOutlined, PlusOutlined, WarningOutlined,
+  LockOutlined, WarningOutlined,
 } from '@ant-design/icons';
-import { Button, Tooltip } from 'antd';
-import axios from 'axios';
+import { Tooltip } from 'antd';
 import { useTheme } from 'styled-components';
-import { Course, UnselectCourses } from 'types/api';
-import { PlannerCourse } from 'types/planner';
-import prepareUserPayload from 'utils/prepareUserPayload';
-import type { RootState } from 'config/store';
+import QuickCart from 'components/QuickAddCartButton';
```

Comment:
rename import to `QuickAddCartButton`. We can technically name our default import anything we want i.e. like above but usually its best to name it as the component name.